site/installation/docker/README.md

```diff
@@ -7,12 +7,24 @@ description: >
     Run kpt in a docker container.
 ---
 
-Use the docker image.
+Use one of the kpt docker images.
 
-[gcr.io/kpt-dev/kpt]
+## [gcr.io/kpt-dev/kpt]
 
 ```shell
 docker run gcr.io/kpt-dev/kpt version
 ```
 
+## [gcr.io/kpt-dev/kpt-gcloud]
+
+An image which includes kpt based upon the Google [cloud-sdk] alpine image.
```

Comment:
Maybe more importantly:

`Image includes `kpt` as well `gcloud` based on ...`

Both images include git right? That's something to note there.

Reply:
I've added a feature list. It's not exhaustive but I tried to include key things you would need: `kpt`, `git`, `gcloud` and the available diff tools.